nservicebus/azure-storage-queues/multi-storageaccount-support.md

```diff
@@ -59,6 +61,28 @@ Each endpoint uses its own Azure storage account, thereby increasing message thr
 
 ![Scale out with multiple storage accounts](azure03.png "width=50%")
```

Comment:
The transition from the above topic to the topic below reads very abruptly, I think the topic below needs a short introduction to make the transition, maybe start with a short introduction here what the problem exactly is (you only describe the solution, not the problem)